code/mob/living/critter/flock/flockdrone.dm

```diff
@@ -638,18 +613,15 @@
 		src.harmedBy(attacker)
 
 /mob/living/critter/flock/drone/attackby(var/obj/item/I, var/mob/M)
-	// check whatever reagents are about to get dumped on us
-	var/has_harmful_chemicals = 0
+	var/has_harmful_chemicals = FALSE
 	if(istype(I, /obj/item/reagent_containers/glass))
 		var/list/reagent_list = I.reagents.reagent_list
 		for(var/reagent_id in reagent_list)
 			var/datum/reagent/current_reagent = reagent_list[reagent_id]
-			// if the reagent mix dumped on us includes a combustible or harmful reagent, the mob has harmful intent
```

Comment:
useful comment

Reply:
I think currently explained by the type check nearby?